cla_backend/apps/legalaid/models.py

```diff
@@ -141,7 +141,7 @@ class PersonalDetails(CloneModelMixin, TimeStampedModel):
     contact_for_research_via = models.CharField(
         max_length=10, default=RESEARCH_CONTACT_VIA.PHONE, choices=RESEARCH_CONTACT_VIA, blank=True, null=True
     )
-    contact_for_research_methods = models.ManyToManyField(ContactResearchMethod, null=True)
```

Comment:
think null=True needs to stay because contact_for_research_methods field was added later and there could be rows that doesn't have contact_for_research_methods